enterprise/admin/__init__.py

```diff
@@ -1255,6 +1255,7 @@ class EnterpriseGroupMembershipAdmin(admin.ModelAdmin):
     """
     model = models.EnterpriseGroupMembership
     list_display = ('group', 'membership_user',)
+    list_filter = ('is_removed',)
```

Comment:
[nit/suggestion] since we're adding in the filter, I think we could also add the same field in the `list_display` to make it clear to the user which list they are viewing.

`list_display = ('group', 'membership_user', 'is_removed')`

Reply:
Updated as per suggestion
<img width="1116" alt="Screenshot 2024-08-20 at 11 26 03 AM" src="https://github.com/user-attachments/assets/b4688b3c-1cfe-4862-9b3c-66e907071048">